### DISCUSSION TITLE
Missions

### DISCUSSION THREAD

**Discussion by TiagoFar78** [TiagoFar78/PrisonEscape#125] [Ideas]
The police team must complete missions in order to get some money to buy useful items. This is a space where everyone can suggest their missions ideas.

**Comment by iquelli** [comment]
- fix toilet (equivalent to filling a cauldron)
- maybe plant something and water it could be a mission
- clean outside area (like remove weed)

**Comment by iquelli** [comment]
- collect x amount of crops?

**Comment by Grandalfwise** [comment]
Cook x amount of food.

**Comment by TiagoFar78** [comment]
Continued in #58
